tachyon/math/finite_fields/prime_field_fallback.h

```diff
@@ -38,7 +38,6 @@ class PrimeField<_Config, std::enable_if_t<!_Config::kIsSpecialPrime &&
 
   using Config = _Config;
   using BigIntTy = BigInt<N>;
-  using MontgomeryTy = BigInt<N>;
   using value_type = BigInt<N>;
```

Comment:
The overlap in the type is quite confusing to me. What's the reason behind this?

Reply:
Do you mean why there exist the type `MontgomeryTy`? If so, I make this type to explicitly denote the return type for `ToMontgomery()`.

Reply:
I was confused on why there were three separate aliases for the same type. Are the values held inside different regardless of type?